rkt/run.go

```diff
@@ -457,14 +457,14 @@ func (e *envFileMap) Set(s string) error {
 			continue
 		}
 		pair := strings.SplitN(scanner.Text(), "=", 2)
-		// Malformed lines
-		if len(pair) != 2 || len(pair[0]) == 0 {
-			return fmt.Errorf("environment variable must be specified as name=value (file %q)", file)
-		}
 		// Skip comments
 		if string(pair[0][0]) == "#" || string(pair[0][0]) == ";" {
 			continue
 		}
+		// Malformed lines
+		if len(pair) != 2 || len(pair[0]) == 0 {
```

Comment:
Yeah, how about we bring back the helper function from the original feature addition? It did this better than the indexing here. Will update tmrw.